build-gluster-org/jobs/centos7-regression.yml

```diff
@@ -6,36 +6,63 @@
     concurrent: true
 
     scm:
-    - git:
-        branches:
-        - $GERRIT_BRANCH
-        refspec: $GERRIT_REFSPEC
-        choosing-strategy: gerrit
-        url: git://review.gluster.org/glusterfs.git
-        wipe-workspace: false
+    - glusterfs
 
     properties:
+    - github:
+        url: https://github.com/gluster/glusterfs
     - discard-after-x:
         x: 90
     - one-build-per-node
 
     triggers:
-    - gerrit:
-        server-name: review.gluster.org
-        trigger-on:
-          - comment-added-contains-event:
-              comment-contains-value: "recheck centos"
-          - comment-added-event:
-              approval-category: Verified
-              approval-value: 1
-        failure-message-file: gluster_regression.txt
-
-        projects:
-          - project-compare-type: 'PLAIN'
-            project-pattern: 'glusterfs'
-            branches:
-              - branch-compare-type: 'ANT'
-                branch-pattern: '**'
+    - github-pull-request:
+        cancel-builds-on-update: true
+        allow-whitelist-orgs-as-admins: true
+        admin-list:
+          - deepshikhaaa
+          - mscherer
+          - csabahenk
+          - jdarcy
+          - itisravi
+          - aravindavk
+          - kalebskeithley
+          - nigelbabu
+          - nixpanic
+          - prashanthpai
+          - pranithk
+          - raghavendrabhat
+          - vbellur
+          - amarts
+          - raghavendrahg
+          - raghavendra-talur
+          - KritikaDhananjay
+          - thotz
+          - rafikc30
+          - humblec
+          - poornimag
+          - ShyamsundarR
+          - anoopcs9
+          - soumyakoduri
+          - spalai
+          - atinmu
+          - kotreshhr
+          - mchangir
+          - nbalacha
+          - mohit84
+          - sunnyku
+          - sanjurakonde
```

Comment:
please add xavi here. Considering he is main maintainer.

Reply:
He was not added to the github's team `gluster-maintainers`. I just added him. Will update this PR